### ISSUE TITLE
Object #<process> has no method 'geteuid'

### ISSUE THREAD

**Issue by utnaf** [doowb/homedir-polyfill#1]
Hi there, I'm deploying an application on an AWS EB environment; by today, during the deploy a gulp task (and then the deploy) fails with this error:

```
 /foobar/node_modules/gulp/node_modules/liftoff/node_modules/findup-sync/node_modules/resolve-dir/node_modules/global-modules/node_modules/global-prefix/node_modules/homedir-polyfill/index.js:37
  var uid = process.geteuid();
  ^
  TypeError: Object #<process> has no method 'geteuid'
  at homedir (/foobar/node_modules/gulp/node_modules/liftoff/node_modules/findup-sync/node_modules/resolve-dir/node_modules/global-modules/node_modules/global-prefix/node_modules/homedir-polyfill/index.js:37:21)
  at Object.<anonymous> (/foobar/node_modules/gulp/node_modules/liftoff/node_modules/findup-sync/node_modules/resolve-dir/node_modules/global-modules/node_modules/global-prefix/index.js:21:14)
  at Module._compile (module.js:456:26)
  at Object.Module._extensions..js (module.js:474:10)
  at Module.load (module.js:356:32)
  at Function.Module._load (module.js:312:12)
  at Module.require (module.js:364:17)
  at require (module.js:380:17)
  at Object.<anonymous> (/foobar/node_modules/gulp/node_modules/liftoff/node_modules/findup-sync/node_modules/resolve-dir/node_modules/global-modules/index.js:11:14)
  at Module._compile (module.js:456:26)
```

The app is deployed on an EC2 machine running Amazon Linux 2016.03 running node 0.10.46.

I noticed that the package global-prefix required `"homedir-polyfill": "^1.0.0"`(starting from yesterday) that is currently tagged as 1.0.1 on this repo, but [on npm](https://www.npmjs.com/package/homedir-polyfill) still at the 1.0.0, which doesn't have the fallback to process.getuid if the process.geteuid is not available, so maybe the package update has not been published yet, or npm got some troubles, or I'm totally wrong.

Hope this helps... Thanks ;)

**Comment by doowb** [comment]
Thanks, for catching this. I published 1.0.1 to npm so a reinstall should fix the issue.

**Comment by utnaf** [comment]
Ok I just tested and it's fixed, thanks!
